lib/network/modules/LayoutEngine.js

```diff
@@ -1281,9 +1273,9 @@ class LayoutEngine {
 
         // we get the X or Y values we need and store them in pos and previousPos.
         // The get and set make sure we get X or Y
-        if (i === 0) {pos = this._getPositionForHierarchy(this.body.nodes[parentId]);}
-        else         {pos = this._getPositionForHierarchy(childNodes[i-1]) + spacing;}
-        this._setPositionForHierarchy(childNode, pos, childNodeLevel);
+        if (i === 0) {pos = this.direction.getPosition(this.body.nodes[parentId]);}
+        else         {pos = this.direction.getPosition(childNodes[i-1]) + spacing;}
```

Comment:
Would prefer this if else to be spread across more lines, rather than 2 lines

Reply:
agree